README.md

```diff
@@ -27,7 +27,7 @@ TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports
 
 - __MySQL Compatible Syntax__
 
-    TiDB acts like it is a MySQL 5.7 server to your applications. You can continue to use all of the existing MySQL client libraries, and in many cases, you will not need to change a single line of code in your application. Because TiDB is built from scratch, not a MySQL fork, please check out the list of [known compatibility differences](./sql/mysql-compatibility.md).
+    TiDB acts like it is a MySQL 5.7 server to your applications. You can continue to use all of the existing MySQL client libraries, and in many cases, you will not need to change a single line of code in your application. Because TiDB is built from scratch, not a MySQL fork, please check out the list of [known compatibility differences](https://github.com/pingcap/docs/blob/master/sql/mysql-compatibility.md).
```

Comment:
I clicked this link on the README page of TiDB master https://github.com/pingcap/tidb, it redirects me to https://www.pingcap.com/docs/sql/mysql-compatibility/
@lilin90 Is this behavior expected?

Reply:
Yes. I think it is expected. @zz-jason